tests/Oryx.Integration.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -129,7 +131,7 @@ public async Task CanBuildAndRunPythonApp_UsingScriptCommandAndSetEnvSwitch()
                .ToString();
             var runScript = new ShellScriptBuilder()
                 .AddDefaultTestEnvironmentVariables()
-                .SetEnvironmentVariable(SettingsKeys.EnableDynamicInstall, true.ToString())
+                .AddCommand($"oryx setupEnv -appPath {appOutputDir}")
```

Comment:
remove this line since you already setup in the dynamic runtime image?

Reply:
Yes, that is right.